.buildkite/pipelines/on_merge.yml

```diff
@@ -78,6 +78,12 @@ steps:
         - exit_status: '*'
           limit: 1
 
+  - command: '.buildkite/pipelines/e2e.yml'
+    label: Trigger E2E tests
```

Comment:
can we call this `Trigger unsupported ftr tests`?

Reply:
Pushed [8187f36](https://github.com/elastic/kibana/pull/152846/commits/8187f367c8057e40cc884e2304768599a6ae4e71)